scripts/testing/city_runner/ui.py

```diff
@@ -108,36 +113,43 @@ def print_results(self, results):
         cts_rate = self.calc_progress(results.num_passes_cts, results.num_total_cts, 1)
         cts_fail_rate = self.calc_progress(results.num_total_cts-results.num_passes_cts,
                                            results.num_total_cts, 1)
+        unknown_pass_rate = self.calc_progress(len(results.unknown_pass_list), results.num_tests, 1)
+        unknown_fail_rate = self.calc_progress(len(results.unknown_fail_list), results.num_tests, 1)
+        unknown_timeout_rate = self.calc_progress(len(results.unknown_timeout_list), results.num_tests, 1)
```

Comment:
Can these go before the calculation of `cts_rate` and `cts_fail_rate`? I was wondering for a moment why they were using `num_tests` rather than `num_total_cts` like the rest.